Framework/Algorithms/src/PDCalibration.cpp

```diff
@@ -272,6 +305,38 @@ void PDCalibration::exec() {
   m_tofMin = tofBinningParams.front();
   m_tofMax = tofBinningParams.back();
 
+  vector<double> tzeroRange = getProperty("TZEROrange");
+  if (tzeroRange.size() == 2) {
+    m_tzeroMin = tzeroRange[0];
+    m_tzeroMax = tzeroRange[1];
+
+    std::stringstream msg;
+    msg << "Using tzero range of " << m_tzeroMin << " <= "
+        << "TZERO <= " << m_tzeroMax;
+    g_log.information(msg.str());
+  } else {
+    g_log.information("Using all TZERO values");
+
+    m_tzeroMin = -1. * std::numeric_limits<double>::max();
```

Comment:
I didn't notice that one in the documentation. I knew that `std::numeric_limits<double>::min()` gave back a small positive number which wasn't what I wanted.